sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
```

Comment:
Fair. I'll check the local repo first for the version and if that fails, the remote repo for now. Once the replace and repo init interactions are figured out we can change the SDK detection behavior too. 

FWIW, it would be good if cork create --replace replaced everything in the chroot to match the new branch and manifest. Maybe even use  --force-sync when doing repo sync.